types/leo-profanity/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for leo-profanity 1.5
+// Project: https://github.com/jojoee/leo-profanity (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
```suggestion
// Project: https://github.com/jojoee/leo-profanity
```

Reply:
I have removed the link in the latest commit